internal/expr/parser.go

```diff
@@ -159,44 +131,22 @@ func (sfc sqlFunctionCall) String() string {
 	return sfc.raw
 }
 
-// inputExpr represents a named parameter that will be sent to the database
-// while performing the query.
-type inputExpr struct {
-	sourceType valueAccessor
-	raw        string
-}
-
-func (p *inputExpr) String() string {
-	return fmt.Sprintf("Input[%+v]", p.sourceType)
-}
-
-func (p *inputExpr) expr() {}
-
-// outputExpr represents a named target output variable in the SQL expression,
-// as well as the source table and column where it will be read from.
-type outputExpr struct {
-	sourceColumns []columnAccessor
-	targetTypes   []memberAccessor
-	raw           string
-}
-
-func (p *outputExpr) String() string {
-	return fmt.Sprintf("Output[%+v %+v]", p.sourceColumns, p.targetTypes)
-}
-
-func (p *outputExpr) expr() {}
-
 // bypass represents part of the expression that we want to pass to the backend
 // database verbatim.
 type bypass struct {
```

Comment:
You relocated the other types, but left this one in parser. Is that deliberate?

Reply:
On reflect it should be with the others. Moved.